src/Core/Manifest/VersionProvider.php

```diff
@@ -180,55 +181,55 @@ public function getModules()
     /**
      * Tries to obtain version number from composer.lock if it exists
      *
-     * @param array $modules
-     * @return array
+     * @param array<string> $modules
+     * @return array<string|string>
      */
     public function getModuleVersionFromComposer($modules = [])
     {
         $versions = [];
-        $lockData = $this->getComposerLock();
-        if ($lockData && !empty($lockData['packages'])) {
-            foreach ($lockData['packages'] as $package) {
-                if (in_array($package['name'], $modules ?? []) && isset($package['version'])) {
-                    $versions[$package['name']] = $package['version'];
-                }
-            }
+        foreach ($modules as $module) {
+            $versions[$module] = \Composer\InstalledVersions::getPrettyVersion($module);
```

Comment:
```suggestion
            $versions[$module] = InstalledVersions::getPrettyVersion($module);
```
Sorry, should have caught this last time. Please use a `use` statement instead.